android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -75,6 +76,10 @@ public WebRTCModule(ReactApplicationContext reactContext, Options options) {
         ThreadUtils.runOnExecutor(() -> initAsync(options));
     }
 
+    public Activity getCurrentActivityHack() {
+        return super.getCurrentActivity();
```

Comment:
`getCurrentActivity` is protected in `super` and I need it in order to acquire AudioManager.